devicon.json

```diff
@@ -3744,16 +3744,25 @@
         ],
```

Comment:
Might want to add `open-source` to the tags. 🙂

Reply:
Done! 👍🏼